### ISSUE TITLE
predict after cross-validation using xgboost [question]

### ISSUE THREAD

**Issue by RanaivosonHerimanitra** [dmlc/xgboost#92]
This is my first trial with xgboost (very fast!). But I'm a little bit confused   .
 In fact, I trained a model using xgb.cv as follows:
xgbmodel=xgb.cv(params=param, data=trainingdata, nrounds=100, nfold=5,showsd=T,metrics='logloss')
Now I want to predict with my test set but xgbmodel seems to be a logical value (TRUE in this case)
How could I predict after cv? Should I use xgb.train then?
HR

**Comment by vikasnitk85** [comment]
Hi,

There is a parameter prediction=TRUE in xgb.cv, which returns the prediction of cv folds. But it is not clear from the document that for which nround, the predictions are returned?
